packages/api/lambdas/process-s3-dead-letter-archive.js

```diff
@@ -64,6 +71,11 @@ async function processDeadLetterArchive({
     }
   } while (listObjectsResponse.IsTruncated);
   /* eslint-enable no-await-in-loop */
+  // Lambda run as an async operation must have a return
+  return {
```

Comment:
Without an actual output returned from the Lambda, I think I was seeing the PG write fail on schema validation. Though now I feel like I need to double check this assertion